roles/cloudman-boot/tasks/storage.yaml

```diff
@@ -10,3 +10,16 @@
     --set storageClass.defaultClass=true
     --set storageClass.reclaimPolicy="Delete"
     --set storageClass.allowVolumeExpansion=true
+
+- name: Helm install RClone for AWS
+  command: >
+    /usr/local/bin/helm install rclone-csi wunderio/csi-rclone
+    --namespace "{{ cm_namespace_name }}"
+    --set storageClass.name="rclone"
+    --set params.remote="s3"
+    --set params.remotePath="{{ rancher_server | replace('.', '-') }}-gvl-data"
+    --set params.s3-provider="aws"
+    --set params.s3-endpoint="https://s3.{{ cm_initial_cluster_data | b64decode | from_yaml).cloud_config.target.target_zone.region.region_id }}.amazonaws.com"
+    --set params.s3-access-key-id="{{ cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.aws_access_key }}"
+    --set params.s3-secret-access-key="{{ cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.aws_secret_key }}"
```

Comment:
What happens if the aws_access_key is None for example? Perhaps we should insert ignore_errors=true?